infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```
@@ -70,6 +70,9 @@ param parAzBastionName string = '${parCompanyPrefix}-bastion'
 ])
 param parAzBastionSku string = 'Standard'
 
+@sys.description('Switch to enable/disable Bastion native client support.')
+param parBastionTunneling bool = false
```

```suggestion
param parAzBastionTunneling bool = false
```
making it match the others